index.html

```diff
@@ -12,14 +12,24 @@
   <body>
     <header id="store">
       <h1>Greengrocers</h1>
+      <div class="filter-buttons">
+        <button class="fruit-filter">Show Fruits</button>
+        <button class="veg-filter">Show Vegetables</button>
```

Comment:
This part seems like it is part of the extensions.... Nice job getting to the extensions